src/python/pl_orquestra/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Auxiliary file required for defining workflow steps when using Orquestra.
```

Comment:
This could be perhaps more descriptive (and describe the module itself)

Reply:
Oh, this is actually a little misleading at this point with adding the code piece by piece! This will be a completely empty subpackage & module, this `__init__` file is just needed so that Orquestra can install the `steps` correctly. More description on the step defined is included in the `expval.py` module docstring in #2.